githubmarkdownui/job.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from githubmarkdownui.emoji import Emoji
+
+
+@dataclass
+class Job:
+    """This class is intended to be used to help with the creation of job lists or job trees, which can be useful
+    for displaying jobs and child jobs within a build pipeline.
+
+    Each job should have an emoji to signal job success (Emoji.CHECK_MARK or Emoji.X), a description of the job,
+    and can optionally have any additional information about the job.
+    """
+    emoji: Emoji
```

Comment:
Is `status` better than `emoji` here ? status/state itself can be also a model that can be mapped to certain emojis.

Reply:
I think emoji is better here as the use case in mind was creating a job tree for CI and displaying emojis for success/failure, unless there is another use case you can think of where something more generic is helpful

Reply:
Talked to Terrence offline, this is a non issue and makes sense for his use case. Will resolve this conversation.